openssl/src/x509/verify.rs

```diff
@@ -58,6 +59,61 @@ bitflags! {
     }
 }
 
+pub struct PurposeId(c_int);
```

Comment:
These seem to fit better in the store module since that's where they're actually used. I'd call them `X509Purpose` and `X509Trust` probably I think.

Reply:
Sure, I originally put them in the verify module since trust and purpose values are also used in `X509_VERIFY_PARAM`, in addition to `X509_STORE`.